### PR TITLE
Upgrade GitHub Actions

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -15,8 +15,10 @@ jobs:
           - os: ubuntu-latest
             arch: aarch64
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
```
Comment (nsoranzo) [inline .github/workflows/deploy.yaml]:
```suggestion
          python-version: '3.x'
```

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -35,16 +37,18 @@ jobs:
...
 
   build_sdist:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
```
Comment (nsoranzo) [inline .github/workflows/deploy.yaml]:
```suggestion
          python-version: '3.x'
```

**Review by nsoranzo — APPROVED** [review]
Thanks!

**Comment by cclauss** [comment]
Flake8 does not like https://github.com/bxlab/bx-python/pull/53

**Comment by nsoranzo** [comment]
> Flake8 does not like #53

Not exactly, E275 has recently been added in pycodestyle 2.9.0 https://pycodestyle.pycqa.org/en/latest/developer.html#id3

If you drop the last commit disabling E275, we can merge this and address the flake8 errors separately.

**Comment by nsoranzo** [comment]
> If you drop the last commit disabling E275, we can merge this and address the flake8 errors separately.

Done in https://github.com/bxlab/bx-python/pull/84 .

**Comment by nsoranzo** [comment]
Thanks @cclauss !

**Pull request merged**